RSDayFlow/RSDFDatePickerView.h

```diff
@@ -222,7 +255,19 @@
  @param view The view whose date was selected.
  @param date The selected date.
  */
-- (void)datePickerView:(RSDFDatePickerView *)view didSelectDate:(NSDate *)date;
+- (void)datePickerView:(RSDFDatePickerView * __nonnull)view didSelectDate:(NSDate * __nonnull)date;
+
+/**
+ Tells the delegate that the user did select a date in RSDFSelectionModeRange.
+ 
+ The date picker view calls this method when the user successfully selects a date in the date picker view.
+ It does not call this method when you programmatically set the selection.
+ 
+ @param view The view whose date was selected.
+ @param startDate The selected start date for range.
+ @param endDate The selected end date for range.
+ */
+- (void)datePickerView:(RSDFDatePickerView * __nonnull)view didSelectStartDate:(NSDate * __nullable)startDate endDate:(NSDate * __nullable)endDate;
```

Comment:
Does startDate/endDate sound confusing since those names are also used for limiting start/end date?